pkg/services/provisioning/dashboards/file_reader.go

```diff
@@ -190,7 +190,7 @@ func (fr *fileReader) saveDashboard(path string, folderId int64, fileInfo os.Fil
 		CheckSum:   jsonFile.checkSum,
 	}
 
-	_, err = fr.dashboardProvisioningService.SaveProvisionedDashboard(dash, dp, fr.Cfg.AllowUiUpdates)
+	_, err = fr.dashboardProvisioningService.SaveProvisionedDashboard(dash, dp, true)
```

Comment:
I only see one usage of `SaveProvisionedDashboard` and that is here. I would remove the allowUiUpdate argument to `SaveProvisionedDashboard` alltogether and let the `SaveProvisionedDashboard` decide that provisioning validation not should happen.

Reply:
Thanks, that is a great change. :)